easel/i18n/de.json

```diff
@@ -16,7 +16,7 @@
   "upload_to_ipfs": "Auf IPFS hochladen",
   "content_id": "Inhalts-ID",
   "tx_receipt": "TX-Empfang",
-  "close": "Fenster schließen",
+  "close": "Schließen",
```

Comment:
This is approved to shorten the string for the screen, it's not full translation.